### DISCUSSION TITLE
Input line is not correctly parsed for some compiler warnings (e.g. undefined references)

### DISCUSSION THREAD

**Discussion by Sebastian-Maier** [James-Yu/LaTeX-Workshop#3150] [Q&A]
**Is your feature request related to a problem? Please describe.**
Latex Warnings like "Reference 'XYZ' on page 8 undefined on input line 275." are currently not parsed in a way that also extracts the corresponding input line from the compiler output/log. Therefore, there is no easy way to navigate to the line that caused the warning by clicking on the warning. (I am unsure whether this is a bug or intended/limitation.)

**Describe the solution you'd like**
Parsing warnings like these in a fashion that also extracts the input line (for easy navigation).

**Describe alternatives you've considered**
I tried different latex variants (besides lualatex) and different build tools/configurations (Makefile, latexmk).

**Additional context**
Typically, I use lualatex and a custom Makefile for the building of the PDF. I am using macOS. Other errors/warnings (like "Float to large by X.Ypt.") seem to be parsed just fine (and they also include the line number for direct navigation to the corresponding input line).

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

**Comment by jlelong** [comment]
And please provide a minimal example to reproduce the issue.

**Comment by tamuratak** [comment]
If any progress, please open a new issue filling out all the information required in the issue template, and providing a minimal example to reproduce the issue.
